frontend/src/pages/signin-page.tsx

```diff
@@ -73,7 +73,7 @@ function signin() {
     <div className="flex-grow cursor-default bg-white py-4 dark:bg-dark-card">
       <div className="m-4 mb-4 flex justify-center">
         <div className="flex w-full items-center justify-center">
-          <h2 className="w-3/4 pl-48 text-center text-lg font-bold text-black dark:text-dark-primary sm:text-xl">
+          <h2 className="w-3/4 pl-2 text-center text-lg font-bold text-black dark:text-dark-primary sm:text-xl">
             Sign in to WanderLust
```

Comment:
This will make the desktop app look bad, pl-2 should be only for the mobile devices

Reply:
Hello @krishnaacharyaa i am attaching image for sign up page 
for mobile
![image](https://github.com/krishnaacharyaa/wanderlust/assets/59639410/d13a7339-6d0e-4dc2-840a-ede4b2094dac)
for ipad
![image](https://github.com/krishnaacharyaa/wanderlust/assets/59639410/24bde1cb-8bd7-4621-a289-208199819e37)
for desktop

Reply:
Hello @krishnaacharyaa please check i have pushed new changes in same commit

Reply:
Hello @krishnaacharyaa i have pushed please check